### PR TITLE
Changes to bump Westeros version, Selection of OCDM adapter and meta-layer revision settings

### PR DESCRIPTION
**Pull request by HaseenaSainul**
1. Westeros bumped to the version used in the RDK
2. OCDM adapter selection updated based on scp or not
3. TREE_REFERENCE value of thunder updated with meta-wpe layer revision hash

### CODE REVIEW

File: recipes-wpe/wpeframework/wpeframework_git.bb [hunk 1]
```diff
@@ -96,13 +92,26 @@ WPEFRAMEWORK_EXTERN_EVENTS ?= " \
...
+
+    layers = (bb.data.getVar("BBLAYERS", d, True) or "").split()
+    for layer in layers:
+        my_layer = layer.split('/')[-1]
+        if my_layer == 'meta-wpe':
+            return base_get_metadata_git_revision(layer, None)
+
+    return "unknown"
+
+WPE_LAYER_REV = "${@getlayerrevision(d)}"
```
Comment (wouterlucas) [inline recipes-wpe/wpeframework/wpeframework_git.bb:106]: This gets the meta-wpe revision to be added to the build config?

Comment (woutermeek) [reply recipes-wpe/wpeframework/wpeframework_git.bb:106]: Yes, so we can see in a build which revision it was build from.

Comment (HaseenaSainul) [reply recipes-wpe/wpeframework/wpeframework_git.bb:106]: Yes, its updating the meta-wpe revision to the TREE_REFERENCE and we can view it from deviceinfo

Comment (wouterlucas) [reply recipes-wpe/wpeframework/wpeframework_git.bb:106]: nice!

**Pull request merged**